examples/quickstarts/cakephp-mysql.json

```diff
@@ -314,76 +324,102 @@
   ],
   "parameters": [
     {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory limit",
```

Comment:
In the UI, we use headline case for input labels like "Memory Limit". I think the templates should do the same.

https://www.patternfly.org/styles/terminology-and-wording/#capitalization